package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "proto3-json-serializer",
+  "version": "0.1.0",
+  "repository": "googleapis/proto3-json-serializer-nodejs",
+  "description": "Support for proto3 JSON serialiazation/deserialization for protobuf.js",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "files": [
+    "build/src"
+  ],
+  "license": "Apache-2.0",
+  "keywords": [
+    "protobufjs",
+    "protobuf.js",
+    "protobuf",
+    "proto3",
+    "json",
+    "serialization",
+    "deserialization"
+  ],
+  "scripts": {
+    "test": "c8 node -r source-map-support/register node_modules/mocha/bin/mocha build/test/unit",
+    "system-test": "mocha build/test/system",
+    "lint": "gts lint",
+    "clean": "gts clean",
+    "compile": "tsc",
+    "fix": "gts fix",
+    "prepare": "npm run compile",
+    "pretest": "npm run compile",
+    "posttest": "npm run lint",
+    "compile-test-protos": "cd test-fixtures/proto && pbjs -t json test.proto > test.json",
+    "docs": "jsdoc -c .jsdoc.js",
+    "docs-test": "linkinator docs",
+    "predocs-test": "npm run docs",
+    "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
+    "prelint": "cd samples && npm link ../ && npm install"
+  },
+  "devDependencies": {
+    "@types/mocha": "^9.0.0",
+    "@types/node": "^14.11.2",
+    "c8": "^7.7.3",
+    "google-proto-files": "^2.4.0",
+    "gts": "^3.1.0",
+    "jsdoc": "^3.6.7",
+    "jsdoc-fresh": "^1.1.0",
+    "jsdoc-region-tag": "^1.3.0",
```

Comment:
Do we need `jsdoc-region-tag` for this library? It handles edge-cases in JSDoc parsing, e.g., snippet region tags.

Reply:
Not sure, I just copied it :)

Reply:
(but I do have one region tag in quickstart.js)